MoMMI/Modules/thread_reminder.py

```diff
@@ -0,0 +1,18 @@
+import logging
+from MoMMI import MChannel
+import time
+
+
+logger = logging.getLogger(__name__)
+
+async def thread_reminder(channel: MChannel) -> None:
+
+    # Send the message
+    await channel.send("**A round has ended.** You can discuss it at https://boards.4chan.org/vg/catalog#s=ss13g. This channel will be closed for 5 minutes.")
+    # And make it closed
+    await channel.close()
+    # For 5 minutes
+    time.sleep(300)
```

Comment:
oh I almost forgot, you need to do `await asyncio.delay()` not `time.sleep()` here or else you'll lock up the whole bot.